accord-core/src/test/java/accord/utils/Gens.java

```diff
@@ -90,6 +91,63 @@ public static <T> Gen<T> oneOf(Map<Gen<T>, Integer> values)
         return rs -> gen.next(rs).next(rs);
     }
 
+    public static <T> OneOfBuilder<T> oneOf()
```

Comment:
> 8) TCM’s getLogEntries method hit an edge case with snapshots where it assumed the API was inclusive, but its exclusive; this caused a gap in epochs

Added this for the log tests as I added property tests there.  This pattern is super common and figured moving out to a builder (similar to the `CommandsBuilder`) would make it easier for users, and leveraged that in the test